tests/unit/app/services/test_notify.py

```diff
@@ -49,30 +49,29 @@ def test_send_magic_link(self, app):
 
     @responses.activate
     def test_send_collection_submission(self, app, factories):
-        collection_schema = factories.collection_schema.build(name="Submission collection")
-        collection = factories.collection.build(
+        submission = factories.submission.build(
             id=uuid.UUID("10000000-0000-0000-0000-000000000000"),
-            collection_schema=collection_schema,
+            collection__name="My test collection",
```

Comment:
Does this `__` syntax let you set properties on a sub factory? 🤯 or am I missing something?